src/ElementQueries.js

```diff
@@ -194,9 +194,12 @@
 
         /**
          * Start the magic. Go through all collected rules (readRules()) and attach the resize-listener.
+         *
+         * @param {HTMLElement} container only elements of the container are considered (document.body if not set)
+         *
          */
-        function findElementQueriesElements() {
-            var query = getQuery();
+        function findElementQueriesElements(container) {
+            var query = (container) ? container.querySelectorAll.bind(container) : getQuery();
```

Comment:
we should move this to `getQuery(container)` since we support jquery selector engine when available, which can be bound to a context/container as well.